src/App.jsx

```diff
@@ -4,27 +4,40 @@ import TodoList from './TodoList';
 
 import './App.css';
 
-function useSemiPersistentState() {
+function App() {
+	function addTodo(newTodo) {
+		setTodoList([...todoList, newTodo]);
+	}
+
+	useEffect(() => {
+		const p1 = new Promise((resolve, reject) => {
+			setTimeout(() =>
+				resolve({ data: { todoList: [] } }, 2000).then(
+					(result) => {
+						setTodoList(result);
+						setIsLoading(false);
+					}
+				)
+			);
+		});
+	});
+
 	const [todoList, setTodoList] = useState(
 		JSON.parse(localStorage.getItem('savedTodoList')) || []
 	);
 
+	const [isLoading, setIsLoading] = "true";
```

Comment:
Here, you're setting the value of `isLoading` to the string "true," not the Boolean `true`. You need to use the Boolean value.